app/controllers/UIController.scala

```diff
@@ -0,0 +1,27 @@
+package controllers
+
+import play.api.data.Form
+import play.api.data.Forms._
+import play.api.i18n.{I18nSupport, MessagesApi}
+import play.api.mvc.{Action, Controller}
+
+import scala.concurrent.ExecutionContext
+
+/**
+  * This controller is mainly responsible for linking the mockups.
+  * @param messagesApi the messagesAPI for internationalisation.
+  * @param executionContext the execution context for asynchronous execution of program logic
+  */
+class UIController(val messagesApi: MessagesApi)(implicit executionContext: ExecutionContext) extends Controller with I18nSupport {
+  val togglTokenForm = Form(single("togglToken" -> text(maxLength = 20)))
+  val projectName = Form(single("projectName" -> text(maxLength = 20)))
```

Comment:
True, this is left from some testing code, and not required at all. Will remove it.